### ISSUE TITLE
KeyError when trying to add a transaction.

### ISSUE THREAD

**Issue by trin5tensa** [trin5tensa/pigjar#527]
When the user does this:
From ‘Register’ menu select ‘All Accounts’
Click ‘Add Transaction’ button

This is what happens:
```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/tkinter/__init__.py", line 1702, in __call__
    return self.func(*args)
  File "/Users/stephenrigden/Documents/Coding/Pigjar Project/pigjar/pigjar/guidialogs.py", line 286, in <lambda>
    self.buttons[name].ttk.configure(command=lambda b=name: command(button_name=b))
  File "/Users/stephenrigden/Documents/Coding/Pigjar Project/pigjar/pigjar/guidialogs.py", line 306, in substantive
    if not self.keep_alive(button_name=button_name):
  File "/Users/stephenrigden/Documents/Coding/Pigjar Project/pigjar/pigjar/registermenu.py", line 237, in keep_alive
    acc_inst_name = self.records[rec_id][0]
KeyError: None
```

**Comment by trin5tensa** [comment]
Same problem as #524

**Comment by trin5tensa** [comment]
Fixed in #537
